deepfence_worker/cronjobs/notification.go

```diff
@@ -389,7 +464,8 @@ func processIntegration[T any](ctx context.Context, task *asynq.Task, integratio
 	}
 	log.Info().Msgf("%s Total Time taken for integration %s: %d", integrationRow.Resource,
 		integrationRow.IntegrationType, time.Since(startTime).Milliseconds())
-	log.Debug().Msgf("Time taken for neo4j_query_1: %d", neo4j_query_1)
+	log.Debug().Msgf("Time taken for neo4j_query_nc: %d", neo4j_query_nc)
+	log.Debug().Msgf("Time taken for neo4j_query_c: %d", neo4j_query_c)
 	log.Debug().Msgf("Time taken for neo4j_query_2: %d", totalQueryTime)
 	log.Debug().Msgf("Time taken for sending data : %d", totalSendTime)
```

Comment:
We don't need any of these, we capture time for every neo4j queries in our system.
The traces are available via jaegger.